### DISCUSSION TITLE
comments reported counter

### DISCUSSION THREAD

**Discussion by Neeeap** [ThioJoe/YT-Spammer-Purge#863] [Ideas]
Add in a number counter of how many comments people have reported daily so that we don't suddenly get an API quota error when doing a scan and use as much as possible.

**Comment by Firecul** [comment]
There was a little discussion in a couple of topics, this might have useful info, you can see some of ThioJoe's thoughts on it at least https://github.com/ThioJoe/YT-Spammer-Purge/discussions/688#discussioncomment-2211285

**Reply by Neeeap** [reply]
I guess ill try the calculator then :|
